docs/cudf/source/cudf_polars/engine_options.md

```diff
@@ -0,0 +1,25 @@
+# GPUEngine Configuration Options
+
+The `polars.GPUEngine` object may be configured in several different ways.
+
+## Parquet Reader Options
+Reading large parquet files can use a large amount of memory, especially when the files are compressed. This may lead to out of memory errors for some workflows. To mitigate this, the "chunked" parquet reader may be selected. When enabled, parquet files are read in chunks with a limit on the amount of memory that is used, at the cost of a small drop in performance.
+
+
+To configure the parquet reader, we provide a dictionary of options to the `parquet_options` keyword of the `GPUEngine` object. Valid keys and values are:
+- `chunked`, indicates is chunked parquet reading is to be used, default True.
```

Comment:
Let's write the defaults here in sentences rather than like docstrings.
```suggestion
- `chunked` indicates that chunked parquet reading is to be used. By default, chunked reading is turned on.
```